### PR TITLE
[client/rest]: Added light rest readme

### PR DESCRIPTION
**Pull request by AnthonyLaw**
added readme docs

### CODE REVIEW

File: client/rest/README_LIGHT.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Light-rest
+
+## Introduction
+
+This is a lightweight version of the REST server. It provides a limited set of endpoints compared to the full API node. The main differences between the light rest and the full API node are:
```
Comment (Wayonb) [inline client/rest/README_LIGHT.md:5]: try to rephrase...maybe something like this:

```
The light version of the REST server provides limited endpoints compared to the full API node. The main differences between the two are:
```

File: client/rest/README_LIGHT.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- The light rest does not use MongoDB or broker services.
+- it communicates with the peer node using a socket connection.
```
Comment (Wayonb) [inline client/rest/README_LIGHT.md:8]: it -> It

Maybe change to:
```
Light REST only communicates with the server using a socket connection.
```

File: client/rest/README_LIGHT.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Purpose
+
+The purpose of the light rest is to reduce server resource usage, making it a cost-effective. it also offers users the ability to delegate harvesting to the peer node.
+
```
Comment (Wayonb) [inline client/rest/README_LIGHT.md:13]: Does Wallet still work? 🤔

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: client/rest/README_LIGHT.md (hunk 3 above)
Comment (AnthonyLaw) [reply client/rest/README_LIGHT.md:13]: technically it should work, but I found out our statistic service didn't pick up `nodePublicKey`, something I need to fix and test it again in desktop wallet.

Comment (Wayonb) [reply client/rest/README_LIGHT.md:13]: I think the functionality available in the wallet will be limited if database and websocket is not available. 🤔
I will that for your testing.

**Review by Wayonb — APPROVED** [review]
👍🏾

**Pull request merged**